packages/pwa-kit-react-sdk/src/ssr/universal/extensibility/utils.ts

```diff
@@ -16,9 +16,14 @@ import {IApplicationExtension} from './types'
  */
 export const applyAppExtensions = (
     App: React.ComponentType,
-    extensions: IApplicationExtension[]
+    extensions: ApplicationExtension[]
 ): React.ComponentType => {
-    const extendAppHocs = extensions.map(({extendApp}) => extendApp).filter(Boolean)
+    const extendAppHocs = extensions
+        // TODO: All Application Extensions configuration objects will extend from a single IApplicationExtensionConfig type so that we
```

Comment:
docs nit: Can you clarify if "IApplicationExtensionConfig" is a new class, or if we are referring to the configuration in a package of ApplicationExtension?

Reply:
This is just a developer hint and will be removed when Vincent works on the tupal configuration work. The idea I'm outlining for him here is that all application extensions should have a type for their configuration that extends some a since shared type. The reason why I think we should do this is because someone might want to disable their extension without having to remove the extension from the extensions array. They can optionally set `enabled: false` in that extensions config object. 

There might be other shared config values, but this is just going to set us up for success in the future.